static/app/components/core/slider/index.chonk.tsx

```diff
@@ -0,0 +1,317 @@
+import {
+  type CSSProperties,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import type {DO_NOT_USE_ChonkTheme} from '@emotion/react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+import {mergeRefs} from '@react-aria/utils';
+
+import {space} from 'sentry/styles/space';
+
+import type {SliderProps} from './index';
+
+const passthrough = (n: number | '') => n;
+
+export function Slider({
+  formatLabel = passthrough,
+  ref,
+  ...props
+}: SliderProps & {ref: React.Ref<HTMLInputElement>}) {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const refs = mergeRefs(ref, inputRef);
+  const step = toNumber(props.step ?? -1);
+  const {value, min, max} = resolveMinMaxValue(props);
+  const [label, setLabel] = useState(value);
+  const initialProgress = getProgress(value, min, max);
+  const filledSteps = useMemo(() => (step === -1 ? -1 : label / step), [step, label]);
+
+  const handleChange = useCallback(
+    (event: Event) => {
```

Comment:
```suggestion
    (event: React.ChangeEvent<HTMLInputElement>) => {
```
I think you can just use this and avoid the cast on line below